packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts

```diff
@@ -93,9 +93,30 @@ async function processDocumentReference({
     const s3Utils = getS3UtilsInstance();
     const { mimeType, decodedBytes } = getMtomBytesAndMimeType(documentResponse, mtomResponse);
     const strippedDocUniqueId = stripUrnPrefix(documentResponse.DocumentUniqueId);
-    const metriportId = idMapping[strippedDocUniqueId];
+    let metriportId = idMapping[strippedDocUniqueId];
```

Comment:
instead of using `let`, can we use const and have a function that returns the end result of all the manipulations that youre introducing, please?